shell/src/main/java/org/apache/accumulo/shell/commands/DropUserCommand.java

```diff
@@ -51,9 +51,7 @@ private void doDropUser(final Shell shellState, final String user, final boolean
     boolean operate = true;
 
     if (!force) {
-      shellState.getWriter().flush();
-      String line = shellState.getReader().readLine(getName() + " { " + user + " } (yes|no)? ");
-      operate = line != null && (line.equalsIgnoreCase("y") || line.equalsIgnoreCase("yes"));
+      operate = shellState.yorn(getName() + " { " + user + " }");
     }
     if (operate) {
```

Comment:
Same opportunity to simplify the expression without the need for the intermediate `operate` variable.